internal/operands/node-labeller/resources.go

```diff
@@ -0,0 +1,295 @@
+	volName := []string{nfdVolumeName}
+	volMountPath := []string{nfdVolumeMountPath}
+	volMount := volumeMounts(volName, volMountPath)
+
+	container := buildContainer(name, image, args, cmd)
+	container.ImagePullPolicy = core.PullAlways
+	container.VolumeMounts = volMount
+	// TODO add the SecurityContext
+	//var boolVal bool = true
+	//container.SecurityContext.Privileged = &boolVal
+	return container
+}
+
+func initContainerKubevirtNodeLabeller() *core.Container {
+	name := "kubevirt-node-labeller"
+	image := common.GetNodeLabellerImage(name)
+	cmd := []string{"/bin/sh", "-c"}
+	args := []string{"if [ ! -e /dev/kvm ] && [ $(grep '\\<kvm\\>' /proc/misc | wc -l) -eq 0 ]; then echo 'exiting due to missing kvm device'; exit 0; fi; if [ ! -e /dev/kvm ]; then mknod /dev/kvm c 10 $(grep '\\<kvm\\>' /proc/misc | cut -f 1 -d' '); fi; ./usr/sbin/node-labeller"}
+	volName := []string{nfdVolumeName}
+	volMountPath := []string{nfdVolumeMountPath}
+	volMount := volumeMounts(volName, volMountPath)
+	envvar := core.EnvVar{
+		Name: "NODE_NAME",
+		ValueFrom: &core.EnvVarSource{
+			FieldRef: &core.ObjectFieldSelector{
+				FieldPath: "spec.nodeName",
+			},
+		},
+	}
+	container := buildContainer(name, image, args, cmd)
+
+	container.VolumeMounts = volMount
+	container.Env = append(container.Env, envvar)
+	// TODO add the SecurityContext
+	// var boolVal bool = true
+	//container.SecurityContext.Privileged = &boolVal
+	return container
+}
+
+func newDaemonSet(namespace string) *apps.DaemonSet {
+
+	//Build the InitContainers
+	initContainers := []core.Container{
+		*initContainerKvmInfoNfdPlugin(),
+		*initContainerKubevirtCpuNfdPlugin(),
+		*initContainerLibvirt(),
+		*initContainerKubevirtNodeLabeller(),
+	}
+
+	//Build the containers
+	containers := []core.Container{
+		*kubevirtNodeLabellerSleeperContainer(),
+	}
+
+	return &apps.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      DaemonSetName,
+			Namespace: namespace,
+			Labels:    commonLabels,
+		},
+		Spec: apps.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: commonLabels,
+			},
+			Template: core.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: commonLabels,
+				},
+				Spec: core.PodSpec{
+					ServiceAccountName: ServiceAccountName,
+					Containers:         containers,
+					InitContainers:     initContainers,
+					Volumes: []core.Volume{
+						{
+							Name: nfdVolumeName,
+							VolumeSource: core.VolumeSource{
+								EmptyDir: &core.EmptyDirVolumeSource{
+									Medium: "",
+								},
+							},
+						},
+						{
+							Name: configMapVolumeName,
+							VolumeSource: core.VolumeSource{
+								ConfigMap: &core.ConfigMapVolumeSource{
+									LocalObjectReference: core.LocalObjectReference{
+										Name: ConfigMapName,
+									},
+								},
+							},
+						}},
+				},
+			},
+		},
+	}
+}
+
+func newSecurityContextConstraint(namespace string) *secv1.SecurityContextConstraints {
```

Comment:
The security context is not reconciled by the operator.

Reply:
Done